src/test/java/org/tikv/raw/RawKVClientTest.java

```diff
@@ -204,6 +204,26 @@ public void testDeadlineBackOff() {
     }
   }
 
+  @Test
+  public void testBackoffTimeout() {
+    int timeout = 2000;
```

Comment:
sleep 2 seconds will slow down the unit test, could you reduce the time to sleep to reduce the test time?